packages/commonwealth/client/scripts/views/pages/CommunityManagement/CommunityProfile/CommunityProfileForm/CommunityProfileForm.tsx

```diff
@@ -36,7 +36,7 @@ import {
 } from './validation';
 
 const CommunityProfileForm = () => {
-  const userOnboardingEnabled = useFlag('userOnboardingEnabled');
+  const userOnboardingEnabled = true; //useFlag('userOnboardingEnabled');
```

Comment:
should this be hardcoded?

Reply:
nope, will fix

Reply:
[done](https://github.com/hicommonwealth/commonwealth/pull/7824/commits/24f9c2c5478a2404c06e84ec77d9ecffe88a8cb0)